research/requirements.txt

```diff
@@ -19,3 +19,13 @@ seaborn==0.7.1
 serenata-toolbox
 tinys3==0.1.11
 tqdm==4.11.2
+wand==0.4.4
+ghostscript==0.4.1
+keras==2.0.4
+tensorflow>=1.2.1
+h5py>=2.7.0
+Pillow>=4.2.1
+opencv-python
```

Comment:
Can you add the version of the libraries that you are using? It helps in case they change something that make it not working ;)

Reply:
Done!